portal/static/js/src/profile.js

```diff
@@ -621,9 +621,37 @@ export default (function() {
                     return  orgTool.isSubStudyOrg(orgId);
                 }).length;
             },
+            /*
+             * subject is ready to take EMPRO assessment
+             */
+            isSubStudyReadyPatient: function() {
+                return this.isSubStudyPatient() && this.getResearchStudyStatus(EPROMS_SUBSTUDY_ID)["ready"];
```

Comment:
@achen2401  The comment says "subject is ready to take EMPRO assessment". As I read it, this checks for sub-study affiliation, consent, and clinician assignment, but does it also check to see if an assessment should be available now?

Reply:
@mcjustin Justin, I believe so, as I have tested and verified.  I believe the api return a "ready" status based on here:  https://github.com/uwcirg/truenth-portal/blob/0ea5a6c86e59b83daae06f2491eb0e04f9d817e0/portal/models/qb_status.py#L530